css-media-queries/meta-viewport.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <!-- your code here 👇 -->
+  <meta name="Viewport" content="width=device-width">
```

Comment:
"viewport" should not be capitalized here, and you want the "initial-scale=1" portion in content, too. You are welcome to copy the element directly from the documentation, most developers don't write it by hand anyway 👍